_maps/shuttles/arrival_delta.dmm

```diff
@@ -178,40 +178,12 @@
 	},
 /turf/open/floor/iron,
 /area/shuttle/arrival)
-"ar" = (
-/obj/machinery/status_display/evac{
-	pixel_x = -32
-	},
-/obj/item/kirbyplants{
-	icon_state = "plant-21"
-	},
-/obj/effect/turf_decal/tile/neutral{
-	dir = 1
-	},
-/obj/effect/turf_decal/tile/neutral{
-	dir = 8
-	},
-/turf/open/floor/iron,
-/area/shuttle/arrival)
```

Comment:
🤨

Reply:
For anyone wondering: I think Github is just doing a funny because the map is fine locally.

![image](https://github.com/tgstation/tgstation/assets/70232195/0227d27e-554b-4bef-a346-b348c31fcd58)